inference-engine/src/transformations/src/transformations/serialize.cpp

```diff
@@ -406,24 +529,25 @@ void ngfunction_2_irv10(pugi::xml_document& doc,
             layer.append_attribute("version").set_value(
                 get_opset_name(node, custom_opsets).c_str());
         }
+
         // <layers/data>
         pugi::xml_node data = layer.append_child("data");
+        std::string node_type_name{node->get_type_name()};
```

Comment:
Do we change the `node_type_name` or it can be constant?

Reply:
it is changed in `visit_exec_graph_node()`